src/smart-components/group/group-table-helpers.js

```diff
@@ -40,44 +42,116 @@ DefaultPlatformPopover.propTypes = {
   bodyContent: PropTypes.string.isRequired,
 };
 
-export const createRows = (isAdmin, data, selectedRows = []) => {
+export const createRows = (isAdmin, data, selectedRows, expanded = []) => {
   const intl = useIntl();
   return data.reduce(
-    (acc, { uuid, name, roleCount, principalCount, modified, platform_default: isPlatformDefault, admin_default: isAdminDefault }) => [
+    (
+      acc,
+      { uuid, name, roleCount, principalCount, modified, expandTableDetails, platform_default: isPlatformDefault, admin_default: isAdminDefault },
```

Comment:
```suggestion
      { uuid, name, roleCount, principalCount, modified, roles, members, platform_default: isPlatformDefault, admin_default: isAdminDefault },
```
expandTableDetails does not exist - so far you have the data persisted under `roles` key (members are not added yet)